ibm/resource_ibm_is_instance.go

```diff
@@ -729,6 +749,23 @@ func instanceCreate(d *schema.ResourceData, meta interface{}, profile, name, vpc
 			ID: &vpcID,
 		},
 	}
+
+	if dHostIdInf, ok := d.GetOk(isPlacementTargetDedicatedHost); ok {
+		dHostIdStr := dHostIdInf.(string)
+		dHostPlaementTarget := &vpcv1.InstancePlacementTargetPrototypeDedicatedHostIdentity{
+			ID: &dHostIdStr,
+		}
+		instanceproto.PlacementTarget = dHostPlaementTarget
+	}
+
+	if dHostGrpIdInf, ok := d.GetOk(isPlacementTargetDedicatedHostGroup); ok {
+		dHostGrpIdStr := dHostGrpIdInf.(string)
+		dHostGrpPlaementTarget := &vpcv1.InstancePlacementTargetPrototypeDedicatedHostGroupIdentity{
+			ID: &dHostGrpIdStr,
+		}
+		instanceproto.PlacementTarget = dHostGrpPlaementTarget
+	}
+
 	if boot, ok := d.GetOk(isInstanceBootVolume); ok {
```

Comment:
Can we set back isPlacementTargetDedicatedHostGroup and isPlacementTargetDedicatedHost in Read

Reply:
instance api does not return these values in POST and GET methods. only instance template api returns it.